report.Rproj

```diff
@@ -21,5 +21,7 @@ PackageBuildArgs: --compact-vignettes=both
 PackageCheckArgs: --as-cran --run-donttest
 PackageRoxygenize: rd,collate,namespace
 
+UseNativePipeOperator: Yes
```

Comment:
I would prefer if we don't set this for all contributors because `|>` is available only from `R 4.1`, while we support as back as `R 3.6`.